sos/report/plugins/microcloud.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2024 Alan Baghumian <alan.baghumian@canonical.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
```

Comment:
We're just changing the guidelines on licensing, so would you update this to `version 2 or later` please

Reply:
Done. Coming.